From 2f45de6bee43c874c9dbc928601ec8754453f41b Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Thu, 4 Jan 2018 09:22:08 +0800
Subject: [PATCH] ARM: dts: add rk3288-rkisp1.dtsi

There are many isp driver in 4.4 kernel, let's add a dtsi
to help switch between them.

Change-Id: Ida1af575b6c64ffec56ad695933dfdf22cdd72c1
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-rkisp1.dtsi | 23 +++++++++++++++++++++++
 1 file changed, 23 insertions(+)
 create mode 100644 arch/arm/boot/dts/rk3288-rkisp1.dtsi

diff --git a/arch/arm/boot/dts/rk3288-rkisp1.dtsi b/arch/arm/boot/dts/rk3288-rkisp1.dtsi
new file mode 100644
index 000000000000..6d1e70d07899
--- /dev/null
+++ b/arch/arm/boot/dts/rk3288-rkisp1.dtsi
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+
+/ {
+	/delete-node/ cif_isp0;
+	/delete-node/ isp;
+
+	isp: isp@ff910000 {
+		compatible = "rockchip,rk3288-rkisp1";
+		reg = <0x0 0xff910000 0x0 0x4000>;
+		interrupts = <GIC_SPI 14 IRQ_TYPE_LEVEL_HIGH>;
+		clocks = <&cru SCLK_ISP>, <&cru ACLK_ISP>,
+			 <&cru HCLK_ISP>, <&cru PCLK_ISP_IN>,
+			 <&cru SCLK_ISP_JPE>;
+		clock-names = "clk_isp", "aclk_isp",
+			      "hclk_isp", "pclk_isp_in",
+			      "sclk_isp_jpe";
+		assigned-clocks = <&cru SCLK_ISP>, <&cru SCLK_ISP_JPE>;
+		assigned-clock-rates = <400000000>, <400000000>;
+		power-domains = <&power RK3288_PD_VIO>;
+		iommus = <&isp_mmu>;
+		status = "disabled";
+	};
+};
-- 
2.35.3

